core/src/main/java/fr/sncf/osrd/conflicts/Conflicts.kt

```diff
@@ -260,20 +263,19 @@ fun mergeMap(resources : HashMap<Set<Long>, MutableList<Conflict>>, conflictType
                 a.time < b.time -> -1
                 else -> 0}
         })
```

Comment:
Instead of calling sort twice, which only works if the sorting algorithm is stable (the second sort may disturb the order set by the first sort) you should only call it once. The compareTo function (or the sorting key) is responsible for defining the order.

To sort a collection using two criteria:
  - compare the two elements using the first attribute. If the two attributes are different, return the difference sign. otherwise, continue
  - compare the two elements using the second criterion, and return the sign of the difference, or 0 if they are the same